pages/getting-started/installing-cypress.md

```diff
@@ -0,0 +1,334 @@
+
+```json
+{
+  "scripts": {
+    "cypress:open": "cypress open"
+  }
+}
+```
+
+Agora você pode chamar o método do seu projeto principal dessa forma:
+```
+npm run cypress:open
+```
+
+… e o Cypress irá abrir direto para você. 
+
+# Ferramentas CLI
+
+Instalando Cypress pela `npm` você também pode acessar vários comandos CLI.
+
+A partir da versão 0.20.0, o Cypress também é um `node_module` totalmente preparado que você pode exigir em seus scripts do Node. 
+
+Você pode [ler mais sobre CLI aqui.](#)
+
+# Avançado
+## Variáveis de ambiente
+
+| Nome                            |                          Descrição                          |
+|---------------------------------|:-----------------------------------------------------------:|
+| `CYPRESS_INSTALL_BINARY`        |     [Destino do binário Cypress baixado e instalado](#)     |
+| `CYPRESS_DOWNLOAD_MIRROR`       | [Baixa o binário Cypress através de um servidor espelho](#) |
+| `CYPRESS_CACHE_FOLDER`          |     [Altera o caminho do cache binário do Cypress ](#)      |
+| `CYPRESS_RUN_BINARY`            |   [Caminho do binário do Cypress em tempo de execução](#)   |
+| ~~CYPRESS_SKIP_BINARY_INSTALL~~ |               use `CYPRESS_INSTALL_BINARY=0`                |
+| ~~CYPRESS_BINARY_VERSION~~      |                use `CYPRESS_INSTALL_BINARY`                 |
+
+## Instalar binário
+
+Usando a variável de ambiente `CYPRESS_INSTALL_BINARY`, você pode controlar como o Cypress é instalado. Para sobrescrever o que foi instalado, você seta `CYPRESS_INSTALL_BINARY` junto com o comando `npm install`.
+
+Isso é útil se você deseja:
+
+- Instalar uma versão diferente do pacote npm padrão.
+```
+CYPRESS_INSTALL_BINARY=2.0.1 npm install cypress@2.0.3
+```
+
+Especificar uma URL externa (para contornar um firewall corporativo).
+```
+CYPRESS_INSTALL_BINARY=https://company.domain.com/cypress.zip npm install cypress
+```
+
+Especificar um arquivo para instalar localmente ao invés de usar pela internet.
+```
+CYPRESS_INSTALL_BINARY=/local/path/to/cypress.zip npm install cypress
+```
+
+Em todos os casos, o fato do binário ter sido instalado a partir de um local personalizado não é salvo no arquivo `package.json`. Cada instalação repetida precisa usar a mesma variável de ambiente para instalar o mesmo binário.
+
+## Pulando a instalação
+
+You can also force Cypress to skip the installation of the binary application by setting CYPRESS_INSTALL_BINARY=0. This could be useful if you want to prevent Cypress from downloading the Cypress binary at the time of npm install.
+```
+CYPRESS_INSTALL_BINARY=0 npm install
+```
+
+Agora o Cypress irá pular as etapas de instalação já que o módulo `npm` está instalado.
+
+# Cache binário
+A partir da versão 3.0, 
+As of version 3.0, Cypress baixa o seu binário correspondente para o cache do sistema global, para que o binário possa ser compartilhado entre projetos. Por padrão, as pastas de cache globais são:
```

Comment:
Oops, Portuguese and English on the same sentence. 😜

Reply:
done